### ISSUE TITLE
org src block faces are not setted

### ISSUE THREAD

**Issue by stardiviner** [balajisivaraman/emacs-one-themes#2]
Here is the screenshot:

![image](https://user-images.githubusercontent.com/378699/40581867-e3507f34-6195-11e8-8f70-769d38a231e8.png)

After check out `one-themes.el` source code, found it use `,comment` face. That's very weird. Also `org-code`, `org-special-keyword` etc is using comment face too.
Don't know whether this is on purpose. But it does not looks good. I think like leuven-theme is better. distinguish org-code and org-block with different color. org-block can use a different background.
Because Org newer version support `org-block-begin-line` and `org-block-end-line` face. So set them too will looks better.
WDYT?

EDIT:
`org-verbatim` face is not fontified too.

**Comment by balajisivaraman** [comment]
Thanks for the issue.

This is one of those cases where I don't use Org Source code blocks at all, so when creating the theme, I could not verify whether I had used the proper colours.

I agree that we have to clearly distinguish between Org Code and Org Blocks. Also thank you for highlighting `org-block-begin-line` and `org-block-end-line`. I did not know about them.

I'll have a look at it and see what I can do to get them looking better.
